futurex_openedx_extensions/dashboard/details/courses.py

```diff
@@ -130,6 +131,12 @@ def get_courses_queryset(
             ).values('course_id').annotate(count=Count('id')).values('count'),
             output_field=IntegerField(),
         ), 0),
+    ).annotate(
+        completion_rate=Case(
+            When(enrolled_count=0, then=Value(0.0)),
+            default=Cast(F('certificates_count'), FloatField()) / Cast(F('enrolled_count'), FloatField()),
```

Comment:
you can avoid `Cast` with this trick:
```suggestion
            default=F('certificates_count') * 1.0 / F('enrolled_count'),
```
we need explicit `Cast` when the internal results are coming from a string, like `"12"` instead `12`. Otherwise, implicit casting using multiplication by `1.0` is safe to do